lib/Zonemaster/Backend/TestAgent.pm

```diff
@@ -126,7 +124,7 @@ sub run {
 
             $counter{ uc $entry->level } += 1;
         }
-    );
+    ) if ( $show_progress );
```

Comment:
Would you consider using the prefix syntax for the `if` statement? For readability.

Reply:
done